data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -2020,6 +2020,333 @@
+      {
+        "collection": [
+          {
+            "item": "sig_40",
+            "prob": 100,
+            "charges": [ 0, 12 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_sig_40" }
+        ],
+        "prob": 10
+      },
+      {
+        "collection": [
+          {
+            "item": "glock_21",
+            "prob": 100,
+            "charges": [ 0, 13 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_glock_21" }
+        ],
+        "prob": 10
+      },
+      {
+        "collection": [
+          {
+            "item": "usp_45",
+            "variant": "usp_45",
+            "prob": 100,
+            "charges": [ 0, 12 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_usp_45" }
+        ],
+        "prob": 2
+      }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_pistols_damaged_extramags_holstered",
+    "//": "Holstered version, necessary for spawning correctly on zombie cop belts, the magazines 'spill' out into the mag pouches on the belt.",
+    "container-item": "holster",
+    "on_overflow": "spill",
+    "subtype": "collection",
+    "items": [ { "group": "police_pistols_damaged_extramags", "prob": 100 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_longarms",
+    "//": "Rifles/Shotguns that US police would have",
+    "subtype": "distribution",
+    "items": [ { "group": "police_carbines", "prob": 80 }, { "group": "police_shotguns", "prob": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_weapons",
+    "//": "A police officer's primary and secondary weapon",
+    "subtype": "collection",
+    "items": [ { "group": "police_longarms", "prob": 100 }, { "group": "police_pistols", "prob": 100 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "SWAT_carbines",
+    "//": "Carbines that a US SWAT team would use",
+    "subtype": "distribution",
+    "entries": [
+      {
+        "item": "modular_m4_carbine",
+        "variant": "modular_m4a1",
+        "prob": 3,
+        "charges": [ 0, 30 ],
+        "contents-group": "SWAT_carbine_mods"
+      },
+      {
+        "item": "modular_m27_assault_rifle",
+        "variant": "modular_h&k416a5",
+        "prob": 17,
+        "charges": [ 0, 30 ],
+        "contents-group": "SWAT_carbine_mods"
+      },
+      { "item": "modular_m16a4", "prob": 30, "charges": [ 0, 30 ], "contents-group": "SWAT_carbine_mods" },
+      { "group": "modular_ar15", "prob": 50, "charges": [ 0, 30 ], "contents-group": "SWAT_carbine_mods" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "SWAT_shotguns",
+    "//": "Shotguns that a US SWAT team would use",
+    "subtype": "distribution",
+    "entries": [
```

Comment:
mossberg 500, 590, and 930 should also be included. I would also say the remington 1100 but we don't have a shorter barrel version.